### PR TITLE
Materials converted to ES6

### PR DESCRIPTION
**Pull request by mvaligursky**

### CODE REVIEW

**Review by willeastcott — APPROVED** [review]

**Comment by mvaligursky** [comment]
still waiting for one more change by @vkalpias - he's converting standard-material.js

**Comment by mvaligursky** [comment]
looks good to me.

File: src/scene/materials/standard-material.js [hunk 1]
```diff
@@ -1172,13 +1160,16 @@ var _defineMaterialProps = function (obj) {
...
     }
 
     obj._propsSet = [];
-};
+}
 
 _defineMaterialProps(StandardMaterial.prototype);
 
+StandardMaterial.TEXTURE_PARAMETERS = standardMaterialTextureParameters;
+StandardMaterial.CUBEMAP_PARAMETERS = standardMaterialCubemapParameters;
```
Comment (willeastcott) [inline src/scene/materials/standard-material.js:1173]: Declare as static in the StandardMaterial class?

Comment (vkalpias) [reply src/scene/materials/standard-material.js]: Done

**Pull request merged**